tools/update_quest_data.py

```
@@ -0,0 +1,439 @@
+            for enemy in stage.get("enemies", []):
+                traits = [t.get("id", t) if isinstance(t, dict) else t for t in enemy.get("traits", [])]
+                enemy_info = {
+                    "svtId": enemy.get("svt", {}).get("id", 0),
+                    "classId": _extract_class_id_from_traits(traits),
+                    "traits": traits,
+                    "deck": "enemy",
+                    "isServant": 1000 in traits,
+                }
+                enemies.append(enemy_info)
+
+        # 获取副本名称（国服需要中文名）
```

Ruff (0.15.11)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)